example/dryrun/branch_cell.hpp

```diff
@@ -31,7 +31,7 @@ struct cell_parameters {
     std::array<unsigned,2> compartments = {20, 2};  //  Compartment count on a branch.
     std::array<double,2> lengths = {200, 20};       //  Length of branch in μm.
 
-    // The number of synapses per cell.
+    // The number of synapses per cel.l
```

Comment:
Shall we blame EMACS?

Reply:
60 years in, EMACS is still the boogeyman scaring tiny programmers. If you're not careful EMACS will come and mess up your code.